### ISSUE TITLE
Clippings/wx: no manager or context menu

### ISSUE THREAD

**Issue by Luke-Baker** [aecreations/clippings#117]
Nightly 58.0a1 20171104100412
Clippings/wx 6.0a4 (as temporary add-on)
All other add-ons disabled

I wanted to test the new version before Firefox 57 launches. Clicking the toolbar button doesn't do anything and there's no context menu entry. The following is the Browser Console output after loading Clippings/wx 6.0a4 as temporary add-on. No further details when clicking the toolbar button.

```1509811555960	addons.xpi	WARN	Please specify whether you want browser_style or not in your options_ui options.
1509811556306	addons.webextension.46f9df23403fc21d92722b9fab50c48ef01d74e3@temporary-addon	WARN	Please specify whether you want browser_style or not in your browser_action options.
Warning: attempting to write 4850 bytes to preference browser.uiCustomization.state. This is bad for general performance and memory usage. Such an amount of data should rather be written to an external file. This preference will not be sent to any content processes.
SecurityError: The operation is insecure.  dexie.js:2
TypeError: e.defineClass is not a function  dexie-observable.js:1:12460
Content Security Policy: The page’s settings blocked the loading of a resource at self (“script-src moz-extension://74451096-0f09-48c2-b655-7e8a9874c948”). Source: call to eval() or related function blocked by CSP.  dexie.js:2
Clippings/wx: It appears that the extension was newly installed.  Welcome to Clippings 6!  background.js:123
Clippings/wx: No user preferences were previously set.  Setting default user preferences.  background.js:179
Clippings/wx: Initializing browser integration...  background.js:194
TypeError: Dexie is not a constructor  background.js:252:18

**Comment by aecreations** [comment]
Thanks for reporting this.  The last error message is very suspect; will investigate further.

**Comment by aecreations** [comment]
Unable to reproduce the error.  I've tried this with Firefox Nightly on both Windows 7 and macOS El Capitan, and installed 6.0 alpha 3 as a temporary add-on.  In both cases, the console doesn't show the above errors, the Clippings context menu appears, and the toolbar button invokes Clippings Manager when clicked.

Which OS are you using?

**Comment by Luke-Baker** [comment]
Thank you for looking into it.

Windows 10 x64. I tried a fresh profile and it works there, but I don't know what modified preferences could cause this. `dom.popup_allowed_events` set to a blank value was my only suspect, but it's not actually a problem.

On an unrelated note, I don't know if you're aware but the manager window has unreadable text underneath "No clippings found." and a closing x below the search box. Likely related to having a custom DPI set in Windows. [Screenshot](https://i.imgur.com/EwlBl4R.png) (I kept getting an error when trying to attach it here)

**Comment by aecreations** [comment]
The unreadable text is actually the toolbar button captions that are (incorrectly) wrapped: the first button should be "+ New Clipping" and the second should be "+ New Folder".  Similarily, the "x" on the search box shouldn't have been wrapped.  It's probably an issue triggered by the screen resolution.  This will be fixed when the Clippings Manager toolbar is completely overhauled for 6.0 beta 1.

Now that the originally-reported problem is no longer reproducible, I will close this issue.  But feel free to reopen if it happens again.

**Comment by Luke-Baker** [comment]
> It's probably an issue triggered by the screen resolution.

No, it's definitely the Windows DPI. It looks fine when mitigated with `layout.css.devPixelsPerPx=1.0`
> Now that the originally-reported problem is no longer reproducible, I will close this issue.

Fair enough. I created a duplicate of my release profile and it works there as well. My Nightly profile is just for testing and I don't mind discarding it.
